lib/perl/Genome/Model/ReferenceAlignment/Report/Summary.t

```diff
@@ -3,6 +3,11 @@
 use strict;
 use warnings;
 
+BEGIN {
+    #use locks local to this test--must be set before Genome is loaded
+    $ENV{XGENOME_SITE_LOCK_DIR} = '/tmp';
+}
+
```

Comment:
To be extra safe, this should probably be something like ```/tmp/$$``` or a real tmpdir (and remove it after the test finishes).  Multiple copies _could_ get scheduled to run on the same machine at the same time.